hydroflow_lang/src/graph/ops/dest_asyncwrite.rs

```diff
@@ -13,6 +13,32 @@ use quote::quote_spanned;
 /// This handles a stream of bytes, whereas [`dest_sink`](#dest_sink) handles individual items of an arbitrary type.
 ///
 /// Note this operator must be used within a Tokio runtime.
+///
+/// ```rustbook
+/// #[tokio::test]
+/// async fn test_dest_asyncwrite() {
+///     use tokio::io::AsyncReadExt;
+///
+///     // Like a channel, but for a stream of bytes instead of discrete objects.
+///     // This could be an output file, network port, stdout, etc.
```

Comment:
That is, every run of bytes that you send into this `dest` is appended to the previously-sent bytes, without delimiters.